README.md

```diff
@@ -1 +1,11 @@
-``` shell
g++ -static lib_sample_use.cpp -std=c++11 -lfastFM -lglog -pthread -I. -L.
```
\ No newline at end of file
+Build:
+
+    > $GCC_MUSL_ROOT/x86_64-linux-musl-g++ -static -std=c++11 lib_sample_use.cpp libfastFM.a libglog.a -o foo
```

Comment:
I mean please try the shared library thing if it's to much effort. However, I'm not sure at the moment if it would solve the problem.